src/main/SortAnalyzer.java

```diff
@@ -257,10 +257,6 @@ private static String checkForSuggestions(Sort sort) {
             suggestions.append("- A warning will pop up every time you select " + sort.getRunSortName() + ". You might want to change its 'unreasonable limit'.\n");
             warned = true;
         }
```

Comment:
Shouldn't the two `if` statements above also be removed? The former since it will now pop up, the latter since its purpose is based on the deleted part. (yeah, can't select the lines above :()